corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py

```diff
@@ -131,6 +131,11 @@ def testOverwriteCache(self):
         self.assertIsInstance(restore_config_cached.get_payload(), CachedResponse)
         self.assertNotIsInstance(restore_config_overwrite.get_payload(), CachedResponse)
 
+        # even cached responses change the sync log id so they are not the same
```

Comment:
> It's bad because then their submissions from different forms will be attributed to the same synclog and the state will diverge from what is on the phone causing the phone to 412.

Ah! that was the lightbulb I needed 💡. thanks, will update with device id